README.md

```diff
@@ -416,6 +422,20 @@ The above would set refresh to 120 seconds (2 minutes), turn off the background
 
 ------
 
+## Multiple Users
+If you want to use Immich Kiosk with multiple users, you can!
+You'll need to create an api key for each user, and add them to the `immich_users_api_keys` field in the `config.yaml` file:
+```yaml
+immich_users_api_keys:
+  john: "****"
+  jane: "****"
+```
+
+Then you can specify which user to use in the URL:
+`https://{URL}?user=john`
```

Comment:
I'm on my phone so I can't test this feature out (and probably won't till after the new year) so what happens if a user sets a user and then a album/person that's not on their account? e.g. `?user=john&album=ALBUM_FROM_JANES_ACCONT`. I guess they just get an error?

Reply:
fixed it so kiosk will try to fetch all the provided albums from the selected user,
only if all the provided albums are not from the selected user, it will show an error.